app/views/income_employment_status/edit.html.erb

```diff
@@ -0,0 +1,32 @@
+<% content_for :header_title, 'Money & Income' %>
+
+<div class="form-card">
+  <header class="form-card__header">
+    <div class="step-section-header">
+      <div class="step-section-header__subhead">
+        Who in your household is currently employed, or has been in the past
+        30 days?
+      </div>
+      <p class="text--help step-section-header__subhead-help">
+        Employment includes temporary or contract jobs. Self employment
+        includes odd jobs, home businesses, online businesses, etc.
+      </p>
+    </div>
+  </header>
+
+  <div class="form-card__content">
+    <%= form_for @step, as: :step, builder: MbFormBuilder, url: current_path, method: :put do |f| %>
+      <% @step.members.each do |member| %>
+        <%= f.fields_for('members[]', member, hidden_field_id: true) do |ff| %>
+          <div class="household-member-group" data-md5="<%= data_md5(header_name(member))%>">
```

Comment:
Would it work if you targeted `[data-member-name="John Doe"]` instead of the hashed string? I think it would?

so 

```
<div class="household-member-group" data-md5="<%= header_name(member) %>">
```